crates/sui-types/src/crypto.rs

```diff
@@ -267,7 +267,21 @@ pub enum PublicKey {
     Secp256k1(Secp256k1PublicKeyAsBytes),
     Secp256r1(Secp256r1PublicKeyAsBytes),
     ZkLogin(ZkLoginPublicIdentifier),
-    Passkey(Secp256r1PublicKeyAsBytes),
+    Passkey(PasskeyPublicKeyAsBytes),
```

Comment:
here we cannot leverage the fixed bytearray from fastcrypto, but to define a new struct.